tests/erc20/test_erc20_mintable_burnable.py

```diff
@@ -0,0 +1,209 @@
+import pytest
+
+ZERO_ADDRESS = '0x0000000000000000000000000000000000000000'
+
+
+@pytest.fixture
+def c(get_contract, w3, bytes_helper):
+    with open('../contracts/erc20/ERC20MintableBurnable.vy') as f:
+        code = f.read()
+    name = bytes_helper("name", 32)
+    symbol = bytes_helper("sym", 32)
+    c = get_contract(code, name, symbol, 0, 0)
+    return c
+
+
+@pytest.fixture
+def get_log_args(get_logs):
```

Comment:
It would be nice to put this in conftest.py for reuse?

Reply:
It seems so, but conftest.py is coped from https://github.com/ethereum/vyper/blob/97ed5fec89947d41fd91f5190c4478ffcadbb9fb/tests/conftest.py
To do PR to vyper repo, it might be necessary to change vyper repo's one.

Reply:
Be animal